Bojzen-I-Mitten/Stort-spelprojekt
# Gameplay/master

Changes to throwing, jumping and tackling.

How to test:
- Set up 2 clients, connect to yourself if you want to test alone.
- Tackling someone should reset your speed.
- Objects are now charged with both LMB and RMB.
- Objects are now thrown on LMB or RMB mouse UP.
- First jump no longer has a movement penalty, 2nd jump still does however.
